src/helics/application_api/CoreApp.hpp

```diff
@@ -26,112 +26,112 @@ class HELICS_CXX_EXPORT CoreApp
     */
-    explicit CoreApp (const std::string &argString);
+    explicit CoreApp(const std::string& argString);
     /** construct from command line arguments parsed as a single string
     @param ctype the type of core to create
     @param argString a merged string with all the arguments
     */
-    CoreApp (core_type ctype, const std::string &argString);
+    CoreApp(core_type ctype, const std::string& argString);
     /** construct from command line arguments parsed as a single string
     @param ctype the type of core to create
     @param coreName the name of the core to use or create
     @param argString a merged string with all the arguments
     */
-    CoreApp (core_type ctype, const std::string &coreName, const std::string &argString);
+    CoreApp(core_type ctype, const std::string& coreName, const std::string& argString);
 
     /** check if the Core is running*/
-    bool isConnected () const;
+    bool isConnected() const;
     /** check if the broker is ready to accept new federates or cores
      */
-    bool isOpenToNewFederates () const;
+    bool isOpenToNewFederates() const;
     /** forceably disconnect the core*/
-    void forceTerminate ();
+    void forceTerminate();
     /** wait for the core to normally disconnect for a certain amount of time*/
-    bool waitForDisconnect (std::chrono::milliseconds waitTime = std::chrono::milliseconds (0));
+    bool waitForDisconnect(std::chrono::milliseconds waitTime = std::chrono::milliseconds(0));
     /** link a publication and input*/
-    void dataLink (const std::string &source, const std::string &target);
+    void dataLink(const std::string& source, const std::string& target);
     /** add a source Filter to an endpoint*/
-    void addSourceFilterToEndpoint (const std::string &filter, const std::string &endpoint);
+    void addSourceFilterToEndpoint(const std::string& filter, const std::string& endpoint);
     /** add a destination Filter to an endpoint*/
-    void addDestinationFilterToEndpoint (const std::string &filter, const std::string &endpoint);
+    void addDestinationFilterToEndpoint(const std::string& filter, const std::string& endpoint);
     /** make connections from a core using a file*/
-    void makeConnections (const std::string &file);
+    void makeConnections(const std::string& file);
     /** get the identifier of the core*/
-    const std::string &getIdentifier () const;
+    const std::string& getIdentifier() const;
     /** get the network address of the core*/
-    const std::string &getAddress () const;
+    const std::string& getAddress() const;
     /** make a query at the core
     @param target the target of the query "federation", "parent", "core","broker" or a specific named object
     @param queryStr the query to make
     @return a string containing the query results
     */
-    std::string query (const std::string &target, const std::string &queryStr);
+    std::string query(const std::string& target, const std::string& queryStr);
```

Comment:
Do we prefer pointer alignment left or right?

Reply:
The VS2019 mouse over types shows it on the right, but I feel like I've seen some compilers with errors that align it to the left.

Reply:
Errors if they are left aligned?   or they just show left aligned in the error message?

Reply:
They just show left aligned in the error message.